src/screens/Analytics/PerformanceMonitor/PerformanceMonitorEntity.res

```diff
@@ -239,9 +254,12 @@ let tableEntity = EntityType.makeEntity(
   ~getHeading,
 )
 
-let getFailureEntity: entity<array<errorObject>> = {
+let getFailureEntity: entity<array<errorObject>, option<string>> = {
   getChartOption: _ => Dict.make()->JSON.Encode.object,
-  getChartData: (~array as _, ~config as _) => [],
+  getChartData: (~args) => {
+    let _ = args
```

Comment:
why 
let _ = args?
can we directly return []

Reply:
because its was giving a warning of unused variables

Reply:
it can be written as
```
  getChartData: (~args as _) => [],
```